### ISSUE TITLE
Washington Data appears off

### ISSUE THREAD

**Issue by OregonCarrot** [COVID19Tracking/issues#139]
The number of cases in the covidtracking data does not match the cases listed in the [DOH of Washington](https://www.doh.wa.gov/Emergencies/Coronavirus).
 examples: 
3/14-
568 cases in the covidtracking data, but 778 in the DOH of Washington data

3/15
642 in the covid tracking data, but 906 cases in the DOH of Washington data

These examples show that it is not just an issue of what time of day the case number is recorded, as Covidtracking's cases on the 15th are less than the DOH of Washington's case number on the 14th.

It appears almost every day has different data between the two sources. Should the Covidtracking data match the DOH of Washington data, as it is very likely to be the most accurate?

**Comment by careeningspace** [comment]
Hello,

Thank you for getting this back to the top of my priority list. I have updated the CTP Washington data to match the data provided by the state. I did find a discrepancy in what is reported - WA's daily Pos/Neg data does not match the Positive data they publish, so I have not resolved the CTP/WA negative data at this time.

Washington is constantly updating its most recent data, so we will need to modify this data from time to time

**Before**
![Screen Shot 2020-04-09 at 1 18 54 PM](https://user-images.githubusercontent.com/43073915/79004551-f6ec3080-7b22-11ea-9413-b0e875b08857.png)

**After**
![Screen Shot 2020-04-10 at 12 10 18 PM](https://user-images.githubusercontent.com/43073915/79005169-4d0da380-7b24-11ea-901d-4879d7919d5d.png)
